### ISSUE TITLE
Robeyr - which collection is that

### ISSUE THREAD

**Issue by MagdaKrzyz** [BetaMasaheft/Dillmann#460]
Could you take a look at Grébaut's entry 

https://betamasaheft.eu/Dillmann/lemma/Lc4ee85c92b1e4cca9895c56e55309bd0.

What is the collection that he mentions: éthiopien Robeyr.

**Comment by DenisNosnitsin1970** [comment]
I could not find it immediately, we will keep this in mind.

**Comment by eu-genia** [comment]
@anaiswion could possibly know. 
dear @anaiswion does this ring a bell?

**Comment by anaiswion** [comment]
Never heard about this name. Any possibility of a misreading ?

**Comment by DenisNosnitsin1970** [comment]
A typing mistake is possible from the side of S. Grebaut, since it is he how refers to the source, probably only once, in his Supplément au Lexicon linguae aethiopicae, see `https://betamasaheft.eu/Dillmann/?mode=none&q=robeyr&id=Lc4ee85c92b1e4cca9895c56e55309bd0`

**Comment by MagdaKrzyz** [comment]
I don't know if only once becsue there is still much ahead of me. 
But if it is a typo then what could be the correct name? 
As far as I can tell Grebaut is very accurate and he does not allow any mistakes in his dictionary. There is errata to the dictionary and this name is not mention. I don't think he would not notice the mistake while proofreading.

**Comment by anaiswion** [comment]
I am currently discussing this with Vanessa Desclaux, the curator in charge of Oriental Christian Mss at the BnF. She suggests that Robeyr is a private collector, as there is no serial number, only a name. Robeyr is the name of a publisher in the 1930-1940s. She'll keep looking!

**Comment by DenisNosnitsin1970** [comment]
Yes, I saw it. This is a small mystery, or maybe a mistake which is now difficult to correct.

**Comment by eu-genia** [comment]
Probably not a mistake, probably as @anaiswion and Vanessa noted, it is this guy 
![image](https://github.com/BetaMasaheft/Dillmann/assets/17987728/57e98c80-6e07-449e-ba2d-f9cb32f5b643)
